README.md

```diff
@@ -1 +1,34 @@
-umbraco-net-core-example
+# [Umbraco 9 (.NET Core) Docker Example](https://github.com/thecogworks/umbraco-core-docker-example) &middot; [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](../LICENSE.md)
+
+An example project with pre-configured Docker container to run and test the latest Umbraco build.
+
+<p align="center">
+  <img src="docs/img/umbraco-and-docker.png" alt="Umbraco + Docker" />
+</p>
+
+## Installation & Usage
+
+1. [Download Docker](https://docs.docker.com/get-docker/) and install it if you don't have it on your machine yet.
+
+2. Set up/change the password for the SA database user specified in _configs/ms.sql.env_ file.
+
+```bash
+SA_PASSWORD=examplepasswordtodb
+ACCEPT_EULA=Y
+```
+
+3. Run the following command in the directory with _docker-compose.yml_ file:
```

Comment:
This is the fast way to work with it using docker-compose. 

Command `docker-compose up -d` will build (first time) and lunch all services.

To run dedicated we can use:  f.e. `docker-compose up -d umbraco-netcore`